src/rpc/blockchain.cpp

```diff
@@ -44,6 +44,9 @@
 #include <memory>
 #include <mutex>
 
+#include <boost/filesystem/operations.hpp>
+namespace boostfs = boost::filesystem;
```

Comment:
If you are going to introduce new Boost Filesystem usage (try and avoid that if possible), it needs to be wrapped in our [Boost Filesystem wrapper](https://github.com/bitcoin/bitcoin/blob/master/src/fs.h#L19).

Reply:
Ah, yes, of course. Will change it!